tools/geojson_simplifier/plotter.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Plots and compares two Geojson files: before and after running simplify.py.
+
+    Typical usage:
+    python3 plotter.py --original_path original-data/geoId-01.geojson
+                       --simplified_path simplified-data/geoId-01-simple.geojson
+"""
+
+from absl import flags
+from absl import app
+import geopandas as gpd
+import matplotlib
+matplotlib.use('TkAgg')
+import matplotlib.pyplot as plt
```

Comment:
Do we need to update the requirements.txt file?

Reply:
Nope. These packages are already included.